.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,15 +19,32 @@ Examples:
 - The *new feature/bugfix* uses *a new data structure/algorithm* to do *X* instead of *Y*.
 --->
 
-#### Additional information
-- Affected modules and functionalities:
+
+
+### Affected modules and functionalities:
 <!--- Describe here what was changed, added, removed. --->
```

Comment:
I think these two points should remain below the first section with the checkboxes.

## Description
< description goes here >

## Category
- [ ] **Bug fix** (*non-breaking change which fixes an issue*)
- [ ] **New feature** (*non-breaking change which adds functionality*)
- [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
- [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
- [ ] **Other** (*e.g. Documentation, Tests, Configuration*)

## Additional information

### Affected modules and functionalities:
<!--- Describe here what was changed, added, removed. --->

### Key points relevant for the review:
<!--- Describe here what is the most important part that reviewers should focus on. --->

## Checklist
...

Reply:
I agree